libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -411,17 +452,26 @@ private bool GetStrictModeFromOptions(IList<string> options)
                     {
                         if (value == "true")
                         {
-                            result = true;
+                            opt.StrictMode = true;
                         }
-                        else if (value == "false")
+                    }
+                    else if (key == replaceNullKey)
+                    {
+                        var pathStr = "${path}";
```

Comment:
this solution have no ideal with multi ${path}
consider using regex to replace the path result, example:
```
private static readonly nullKeyReplaceStrRegex= new Regex(@"\${path}");
 opt.NullSubstitution = (path) => regex.Replace(value, path);
```

Reply:
fixed